cmd/frontend/registry/client/client.go

```diff
@@ -72,6 +72,14 @@ func getBy(ctx context.Context, registry *url.URL, op, field, value string) (*Ex
 	return x, nil
 }
 
+func GetFeaturedExtensions(ctx context.Context, registry *url.URL) ([]*Extension, error) {
+	var x []*Extension
+	if err := httpGet(ctx, "registry.GetFeaturedExtensions", toURL(registry, path.Join("extensions", "featured"), nil), &x); err != nil {
```

Comment:
```suggestion
	if err := httpGet(ctx, "registry.GetFeaturedExtensions", toURL(registry, "extensions/featured", nil), &x); err != nil {
```

We don't have any variable being joined here, is it worth calling the `path.Join`> 😄